proposals/NNNN-unlock-existential-types-for-all-protocols.md

```diff
@@ -105,9 +105,9 @@ func callMethod(p: P) {
 }
 ```
 
-Evidently, an associated type or other unfortunate requirement (such as `bar`) cannot speak for the rest of the interface. Some protocols still have a useful subset of functionality that either doesn't rely on `Self` and associated types, or relies on them in an existentially compatible way.
+Evidently, an associated type or other unfortunate requirements (such as `bar`) cannot speak for the rest of the interface. Some protocols still have a useful subset of functionality that either doesn't rely on `Self` and associated types, or that relies on them in a way compatible with existential types.
 
-The case of an associated type requirement that is known to have a predefined implementation reveals another downside to the status quo:
+The case of an associated type requirement that is known to have a predefined implementation reveals another downside of the status quo:
```

Comment:
I do not have strong feelings about this, but "of" reads as directed towards the existence of status quo, while "to" –  as directed towards some property of the status quo.

Reply:
Reverted.